chain/dev-v3substrate/config.toml

```diff
@@ -0,0 +1,47 @@
+[global]
+basepath = "~/.gossamer/dev"
+log = "info"
```

Comment:
I'd suggest you to interactively rebase your branch to have 2 commits:

1. File movements
2. File modifications/additions etc.

Such that we can rebase and merge to track file movements.

Reply:
Dan, already overwrote the `gssmr` and `dev` chains in his commit that this branch is based on.  So unless I get rid of the whole history, it won't track the file movement.

Reply:
Ah ok 😢